custom/icds/location_reassignment/dumper.py

```diff
@@ -140,3 +145,63 @@ def _old_location_ids_by_site_code(self):
             for loc in
             SQLLocation.active_objects.filter(domain=self.domain, site_code__in=self.old_site_codes)
         }
+
+
+class HouseHolds(object):
+    valid_operations = [SPLIT_OPERATION, EXTRACT_OPERATION]
+    headers = ['Name of AWC', 'Name of Household', 'Date of Registration', 'Religion',
+               'Caste', 'APL/BPL', 'Number of Household Members', 'Household Members']
+
+    def __init__(self, domain):
+        self.domain = domain
+
+    def dump(self, transitions):
+        """
+        :return: excel workbook with one tab with title as old location's site code,
+        which holds details all household cases assigned to it
+        """
+        rows = {}
+        for operation, details in transitions.items():
+            if operation in self.valid_operations:
+                rows.update(self._get_rows_for_location(operation, details))
+        if rows:
+            stream = io.BytesIO()
+            rows = [(k, v) for k, v in rows.items()]
```

Comment:
Isn't this the same as `rows = rows.items()`?

Reply:
https://github.com/dimagi/commcare-hq/pull/27134/commits/a9370dde45443e42086ee3e4cff5feaba2618bf0